packages/ember-container-query/package.json

```diff
@@ -95,6 +95,7 @@
     "@types/ember__service": "^4.0.9",
     "babel-plugin-ember-template-compilation": "^2.2.5",
     "concurrently": "^8.2.2",
+    "ember-source": "^5.9.0",
```

Comment:
Ideally, I think `ember-element-helper` would remove `ember-source` from `peerDependencies`, because I could get `docs-app` to run by uninstalling `ember-element-helper`.